climate-data-ca/functions.php

```diff
@@ -162,6 +162,20 @@ function child_theme_enqueue()
         wp_enqueue_script('archive-functions');
 
     }
+    
+    if ( is_page ( 'news' ) ) {
```

Comment:
This won't work in French

Reply:
replaced `is_page` function with `is_page_template`